### PR TITLE
Update with added options for multiple event types and emulated key hold events.

### PR DESCRIPTION
**Pull request by bendavid**
Corresponding to https://github.com/home-assistant/home-assistant/pull/27761

### CODE REVIEW

File: source/_integrations/keyboard_remote.markdown [hunk 1]
```diff
@@ -22,10 +22,25 @@ keyboard_remote:
...
-device_description:
+emulate_key_hold:
+  description: Emulate key hold events when key is held down.  (Some input devices do not send these otherwise.)
+  required: false
+  type: boolean
+  default: false
+emulate_key_hold_delay:
+  description:  Number of milliseconds to wait before sending first emulated key hold event
+  required: false
+  type: positive_int
```
Comment (frenck) [inline source/_integrations/keyboard_remote.markdown]: `positive_int` is not a valid type for the documentation, use `integer` instead.

**Review by frenck — CHANGES_REQUESTED** [review]
Please make sure the build passes.

File: source/_integrations/keyboard_remote.markdown (hunk 1 above)
Comment (bendavid) [reply source/_integrations/keyboard_remote.markdown]: This has been changed to float in the end.

**Comment by bendavid** [comment]
I believe all of the previous comments have been addressed now.

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Review by frenck — APPROVED** [review]

**Pull request merged**